pkg/odo/cli/component/create.go

```diff
@@ -601,7 +601,6 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 					co.devfileMetadata.devfileSupport = true
 					co.devfileMetadata.devfileLink = devfileComponent.Link
 					co.devfileMetadata.devfileRegistry = devfileComponent.Registry
-					break
```

Comment:
Why remove the break from here?

Once we've found a matching devfile name, we should exit out of the for loop.

Reply:
The reason is to override the existing match to ensure we always create the latest added registry given the latest added registry is always comes after due to the data structure, but I change the way we store the latest added to the data structure in my latest commit, so we can add `break` back now.

Reply:
Ah, gotcha